packages/react-instantsearch/src/widgets/SearchBox.tsx

```diff
@@ -28,12 +28,19 @@ export type SearchBoxProps = Omit<
      * @default true
      */
     searchAsYouType?: boolean;
+    /**
+     * Whether to update the search state in the middle of a
+     * composition session.
+     * @default false
+     */
+    ignoreCompositionEvents?: boolean;
```

Comment:
This describes what happens when this is `false` right ? I think this should be more clear as I got confused

Reply:
I understood "ignoring composition events" as "I want to send a request no matter if it's composing or not" which seems to actually be the contrary

Reply:
I get what you mean. It's hard not to get too verbose with the impact this option has. I had other names:
- `skipCompositionEvents` (but I guess the problem is similar)
- `updateQueryWithCompositionEvents` (really long but more meaningful?)

Do you have other ideas?

In the end, this option will mostly be surfaced through documentation and guides, so maybe we can go with an acceptable name instead of a more descriptive but really long one.

Reply:
I prefer the second one as it's positive but we would have to change the logic :/
I guess we can keep it as is then, makes sense as it would probably be rarely set